test/e2e/framework.go

```diff
@@ -1219,6 +1221,12 @@ func (b *PodBuilder) WithMutateFunc(f func(*corev1.Pod)) *PodBuilder {
 	return b
 }
 
+func (b *PodBuilder) WithResources(ResReqs, ResLimits corev1.ResourceList) *PodBuilder {
```

Comment:
please use `ResourceRequests` and `ResourceLimits` as parameter names

Reply:
Taken care at both the places. please check.